composer.json

```diff
@@ -23,9 +23,9 @@
         "symfony-cmf/testing": "~1.2,>=1.2.7",
         "doctrine/phpcr-odm": "~1.3",
         "matthiasnoback/symfony-dependency-injection-test": "~0.6",
-        "matthiasnoback/symfony-config-test": "0.*",
+        "matthiasnoback/symfony-config-test": "^1.3.1",
         "phpunit/php-code-coverage": "@stable",
-        "sonata-project/doctrine-phpcr-admin-bundle": "~1.1",
+        "sonata-project/doctrine-phpcr-admin-bundle": "^1.1|^2.0",
```

Comment:
we should not allow sonata admin 2.0 as per https://github.com/symfony-cmf/RoutingBundle/pull/325#issuecomment-166959786